Automattic/newspack-blocks
# feat: update subscribe patterns to use Newspack block

### All Submissions:


### Changes proposed in this Pull Request:

This PR updates the 10 Newsletter Subscribe patterns to use the Newspack Subscribe block. It also tweaks the column widths in pattern 10 to avoid the block wrapping over two lines without having to change it with CSS.

Question: Should we move these patterns to the Newsletter plugin, since they now rely on a block that lives in that plugin? The old version of the pattern required Jetpack to be installed, so it wasn't without dependencies, but (unlike Jetpack!) we have the power to add patterns to the Newsletters plugin.

See 1200550061930446-as-1205041142538659

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Add each of the ten patterns from the Newsletter Subscription section, and confirm that there are no display issues or errors on the front-end, or in the editor.

### Other information:
